.github/workflows/main.yml

```
@@ -154,6 +160,9 @@ jobs:
   # 4. Adjust website restrictions for Firebase Key "Sharezone Web Key".
   web-preview:
     runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write # for FirebaseExtended/action-hosting-deploy to comment on PRs
+      checks: write # for FirebaseExtended/action-hosting-deploy to comment on PRs (without write permissions for checks the action doesn't post a comment to the PR)
```

```suggestion
      checks: write # for FirebaseExtended/action-hosting-deploy to comment on PRs (without write permissions for checks the action doesn't post a comment to the PR, we don't know why)
```

I think it's better to make it clear we don't know why so that in the future one might be more inclined to investigate than see the `checks` permission as a given that can't be changed (for all we know right now it might be a bug)